javalib/src/main/scala/java/lang/System.scala

```diff
@@ -82,4 +82,31 @@ object System {
       stdio.fwrite(buf, 1, 1, stream)
     }
   }
+
+  private class CStdinStream() extends java.io.InputStream {
+    def read(): CInt = {
+      val buffer = stackalloc[scala.Byte](1)
+      var nread  = unistd.read(0, buffer, 1)
+      if (nread < 1)
+        -1
```

Comment:
That's right.
I think that because we read stdin we don't have to handle `EAGAIN` and `EWOULDBLOCK` errors. We would only get them if we set stdin non_blocking, which we don't. Don't you think so?